macosx/HBUtilities.m

```diff
@@ -276,6 +298,23 @@ + (NSString *)automaticNameForSource:(NSString *)sourceName
             [formatter setTimeStyle:NSDateFormatterShortStyle];
             [name appendString:[formatter stringFromDate:date]];
         }
+        else if ([formatKey isEqualToString:@"{Creation-Date}"])
+        {
+            NSString *dateString = nil;
+            NSDateFormatter *formatter = [[NSDateFormatter alloc] init];
```

Comment:
Maybe we can cache the formatter as a global var somewhere instead of recreating it each time.

Reply:
- [ ] cache this object out to share across the class